app/forms/password_form.rb

```diff
@@ -11,7 +11,6 @@ def submit(params)
     @password = params[:password]
     @password_confirmation = params[:password_confirmation]
     @request_id = params.fetch(:request_id, '')
-    @confirmation_enabled = params[:confirmation_enabled].presence
```

Comment:
There's a reference to this in the spec, should it be removed as well?

https://github.com/18F/identity-idp/blob/e7e88a29ed4fc3487e2495e7640714100a0d60c8/spec/forms/password_form_spec.rb#L24

Reply:
Good catch, updated.